crates/router/src/routes/payments.rs

```diff
@@ -593,6 +593,13 @@ pub async fn payments_connector_session(
     let flow = Flow::PaymentsSessionToken;
     let payload = json_payload.into_inner();
 
+    let header_payload = match HeaderPayload::foreign_try_from(req.headers()) {
+        Ok(headers) => headers,
+        Err(err) => {
```

Comment:
Previously they were using default right, let's use the same instead of throwing error